pkg/network/ebpf/c/protocols/tls/https.h

```diff
@@ -55,6 +56,12 @@ static __always_inline void classify_decrypted_payload(protocol_stack_t *stack,
     // Protocol is not HTTP/HTTP2/gRPC
     if (is_amqp(buffer, len)) {
         proto = PROTOCOL_AMQP;
+        goto update_stack;
+    }
+
+    if (is_redis(buffer, len)) {
+        proto = PROTOCOL_REDIS;
+        goto update_stack;
     }
```

Comment:
does it make sense to use goto instead of if-else?
I'd say - check complexity of the program, and use the leaner solution

Reply:
The generated code is identical with both methods. But I'll change it, at first it seemed like it could be a good idea, but using goto ends up being more code to express the same thing.